### ISSUE TITLE
I am not able to access the gui of radarr,sonarr,qbittorrent, or jackett when it is under NordLynx

### ISSUE THREAD

**Issue by BhadrinathKS** [bubuntux/nordlynx#144]
##### :warning: Make sure to follow the template, troubleshoot on your own first, review [Open/Closed Issues](https://github.com/bubuntux/nordlynx/issues), [Discussions](https://github.com/bubuntux/nordlynx/discussions), [Wiki](https://github.com/bubuntux/nordlynx/wiki) and consider creating a discussion thread instead. :warning:

I recently installed the new Nordlynx container and after turning the container on adding all the *arr apps, I am unable to access the gui on any of them.

Docker Compose:
version: "3"
services:
  NordLynx:
    image: ghcr.io/bubuntux/nordlynx
    hostname: NordLynx
    container_name: NordLynx
    cap_add:
        - NET_ADMIN
    environment:
        - PRIVATE_KEY=[redacted]
        - QUERY=filters\[servers_groups\]\[identifier\]=legacy_p2p
        - NET_LOCAL=10.0.50.0/24
        - ALLOWED_IPS=0.0.0.0/0
        - TZ=America/Chicago
    ports:
        - 8080:8080
        - 7878:7878
        - 8989:8989
        - 9117:9117
    sysctls:
        - net.ipv4.conf.all.src_valid_mark=1
        - net.ipv4.conf.all.rp_filter=2
        - net.ipv6.conf.all.disable_ipv6=1
  qbittorrent:
    container_name: QB
    image: wernight/qbittorrent:latest
    volumes:
        - /Inf1/NORD/Downloads:/data/downloads
        - /Inf1/NORD/Config:/Config
    network_mode: service:NordLynx
    depends_on:
        - NordLynx
  radarr:
    image: lscr.io/linuxserver/radarr:latest
    container_name: Radarr
    environment:
        - PUID=0
        - PGID=0
    volumes:
        - /Docker/Radarr/Config:/config
        - /Inf1/Media:/data/media
        - /Inf1/NORD/Downloads:/data/downloads
    restart: unless-stopped
    network_mode: service:NordLynx
  sonarr:
    image: lscr.io/linuxserver/sonarr:latest
    container_name: Sonarr
    environment:
        - PUID=0
        - PGID=0
        - TZ=Etc/UTC
    volumes:
        - /Docker/Sonarr/Config:/config
        - /Inf1/Media:/data/media
        - /Inf1/NORD/Downloads:/data/downloads
    restart: unless-stopped
    network_mode: service:NordLynx
  jackett:
    image: lscr.io/linuxserver/jackett:latest
    container_name: Jackett
    environment:
        - PUID=0
        - PGID=0
    volumes:
        - /Docker/Jackett/Config:/config
        - /Inf1/NORD/Downloads:/downloads
    restart: unless-stopped
    network_mode: service:NordLynx
```
version: '3'
services:
  vpn:
    image: bubuntux/nordlynx


### **Expected behavior**
I need to be able to access the GUIs' of the apps that are under the vpn

### **Logs**
No logs indicating any problems, all of the containers start and work as expected, just cannot access the GUI

**Comment by abdulwahid1st** [comment]
yes it sucks.

the way I work around this issue is by forwarding local port on the docker host machine, by either ssh local tunneling or using reverse proxy.

**Comment by cwdbutler** [comment]
hey, i think the problem might be that your `QUERY` variable has syntax errors. i was able to see them in my logs on portainer.
it should be:
```
- QUERY=filters\[servers_groups\]\[identifier\]=legacy_p2p
```

**Comment by jbrown1987** [comment]
I'm seeing the same issue on my setup.

**Issue:**
Deluge GUI not showing and unable to start the local deamon when connected behind nordvpn wireguard using the latest build of /bubuntux/nordlynx.

Has only happened recently (last few days) on the latest build of the nordvpn container image:
bubuntux/nordlynx:latest 
build_version=Linuxserver.io version:- 5d487e0d-ls24 Build-date:- 2023-05-27T13:29:41+00:00

**Expected result:**
Able to access the deluge GUI when using the nordvpn container as the network.

**Steps already tried**
1. I've tried adding the query cwdbutler mentioned = Did not resolve the issue
2. Check deluge container GUI works outside of nordvpn = Working as expected

**Other relevent info:**
My logs for the deluge don't show any issues (as far as I can see). 

The only issue I can find is when inspecting the GUI page on load I see the following on the network tab:
Network Get = NS_ERROR_NET_RESET

**Comment by brum74** [comment]
Double check your NET_LOCAL subnet, you should list the subnet(s) for computers that you want to allow access.  So if your computer you are connecting from is not in 10.0.50.0/24 then that is most likely your issue.

**Comment by landonlewis** [comment]
@jbrown1987 I am experiencing this issue as well.  It started when I upgraded Docker Desktop to 4.24.  I downgraded to 4.19 (the version I had prior) and it started to work again.  I believe this is a Docker issue.

**Comment by jfishie** [comment]
I am also seeing this behaviour. Killing the VPN container to test functionality works but GUI for containers that depend_on nordlynx do not come back without manual restart.

**Comment by jfishie** [comment]
Changing the network_mode from service:nordlynx to container:nordlynx worked for me. I am now able to access the web GUI's if there is a change / restart of the nordlynx container. 

I do not know the pros or cons of using container vs service but it worked for me.

**Comment by tippfehlr** [comment]
@BhadrinathKS your local network probably doesn’t is `NET_LOCAL=10.0.50.0/24`. Is this resolved when providing `NET_LOCAL=192.168.0.0/16` instead?

**Comment by michael-pptf** [comment]
When opening ports in the nordlynx container, open the ports with your HOST IP like so:
`ports:
- [HOST_IP]:8080:8080
- [HOST_IP]7878:7878
- [HOST_IP]8989:8989
- [HOST_IP]9117:9117`

eg:
- 192.168.1.100:8080:8080
